models/zone/src/zone-draft/presets/sample-data-fashion/index.ts

```diff
@@ -0,0 +1,11 @@
+import withCountryAustralia from './with-country-australia';
+import withCountrySpainAndGermany from './with-country-spain-and-germany';
+import withCountryUsa from './with-country-usa';
```

Comment:
No `with` prefix per Slack discussion (since these are highly specific to the sample data)

Reply:
I like it. `countryUsa` is sufficient then? And any ideas about a better name for `countrySpainAndGermany`?

Reply:
Yes and I don't think there's a better name